Firestore/core/src/firebase/firestore/local/local_serializer.cc

```diff
@@ -137,11 +156,8 @@ firestore_client_NoDocument LocalSerializer::EncodeNoDocument(
 
 std::unique_ptr<NoDocument> LocalSerializer::DecodeNoDocument(
     Reader* reader, const firestore_client_NoDocument& proto) const {
-  if (!reader->status().ok()) return nullptr;
```

Comment:
This and below are the checks I talked this morning. I guess now the check for status().ok() is centralized somewhere and thus individual deserializer does not check it anymore. Just want to confirm.